src/Uno.UI/UI/Xaml/Internal/InputManager.Keyboard.skia.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using Uno.Foundation.Extensibility;
+using Uno.Foundation.Logging;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Input;
+
+namespace Uno.UI.Xaml.Core;
+
+partial class InputManager
+{
+	internal KeyboardManager Keyboard { get; private set; } = default!;
+
+	partial void ConstructKeyboardManager() => Keyboard = new(this);
```

Comment:
You can remove the `default!` and add `[MemberNotNull(nameof(Keyboard))]` to `CosntructKeyboardManager`.

Reply:
Applied the suggestions to the next PR (DragDrop manager)